Database/seed.js

```diff
@@ -35,7 +37,14 @@ const seed = () => {
       image.user = faker.name.firstName() + faker.name.lastName();
       image.hotel = hotelName;
       image.tag = tags[tagsNum];
-      image.special = bool;
+      specialObj.is = bool;
+
+      if (bool) {
+        specialObj.specialItem = gamble > .5 ? `https://cdn.wallpapersafari.com/99/98/2IjALn.jpg` : `https://www.tripadvisor.com/Hotel_Review-g150807-d152680-Reviews-Fiesta_Americana_Condesa_Cancun_All_Inclusive-Cancun_Yucatan_Peninsula.html#/media/152680/316101332:v/?albumid=14&type=0&category=14`;
```

Comment:
This reminds me that I need to use ternary operators more often!

Reply:
Be careful with that. Annah has said in the past that people shouldn't use them very often, but I like them so I do... They work well with some react issues, so it's always good to have in one's back pocket.